### DISCUSSION TITLE
Why you choose unreal.js over PuerTS?

### DISCUSSION THREAD

**Discussion by friuns2** [hiperbou/kotlin-unreal#23] [General]
Awesome project! I seen your other unity project is using PuerTS, Why you choose unreal.js over PuerTS?

Does this project converts kotlin to js?

**Comment by hiperbou** [comment]
This project started in 2017 while the first commit in PuerTS is from 2020 according to its github history, so it didn't exist at the time.
I haven't evaluated PuerTS with Unreal, so I couldn't tell if there's any benefit over unreal.js.

Kotlin is capable of compiling to Javascript so I used this feature to interoperate with unreal.js, so basically it's the same than using Javascript but with the benefits of the Kotlin language.
